researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/ResearchStackDAO.java

```diff
@@ -0,0 +1,28 @@
+package org.sagebionetworks.bridge.researchstack;
+
+import android.content.Context;
+
+import org.researchstack.backbone.model.User;
+import org.sagebionetworks.bridge.android.manager.dao.SharedPreferencesJsonDAO;
+
+/**
+ * Created by jyliu on 2/9/2017.
+ */
+
+public class ResearchStackDAO extends SharedPreferencesJsonDAO {
```

Comment:
I agree this needs further refinement. I do think translation is appropriate like you mention. The android-sdk should work with Bridge models only, and Bridge models shouldn't and don't make it outside of DataProvider. 

However, a simple adapter didn't work in this case because the Bridge models don't contain enough information to create the User object in RS, hence an additional storage in researchstack-sdk. It seemed all l I needed in addition to Bridge models was birthday, but the use case for User was unclear to me. So I put in this short term fix of an additional storage layer for things needed by RS but unknown to Bridge to not break functionality